react/features/conference/components/web/WebLoginButton.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import { createDeepLinkUrl } from '../../../base/util/createDeepLinkUrl';
+
+const signDeepLink = () => {
+    const currentUrl = new URL(window.location);
+
+    currentUrl.search = '';
+
+    const signLink = createDeepLinkUrl({
+        type: 'sign-message',
+        message: `I would like to generate JWT token at ${new Date().toUTCString()}`,
+        'x-success': `${currentUrl}?address={address}&result=success&signature={signature}`
+    });
+
+    return createDeepLinkUrl({
+        type: 'address',
+        'x-success': signLink
```

Comment:
I can't use deeplinks in such way. I need to back to jitsi first, save address and forward to sign

Reply:
@mradkov It will be nice if `sign-message` method can forward with address too, in this case I can use the current implementation and I don't need to back to jitsi and save adress to local storage. Also for sign method already knows about address, so maybe it can back it with params. I will add one more issue and implement it with localsStorage at this time.